support/mender-inventory-hostinfo

```diff
@@ -17,5 +17,7 @@ cat /proc/meminfo | awk '
 /MemTotal/ {printf("mem_total_kB=%d\n", $2)}
 '
 
-echo "hostname=$(cat /etc/hostname)"
-
+hostname="localhost"
+hostname >/dev/null 2>&1 && hostname="$(hostname)"
+[ "$hostname" == "" ] && [ -f /etc/hostname ] && hostname="cat /etc/hostname 2>/dev/null"
```

Comment:
small misspell with the double quotes, the above has to read:

```
[ "$hostname" == "" ] && [ -f /etc/hostname ] && hostname=$(cat /etc/hostname 2>/dev/null)
```

Reply:
Changes are ready